web/annotation_layer_builder.css

```diff
@@ -140,7 +140,8 @@
    * area, causing horizontal scrolling. We avoid this by extending the width
    * when the element has focus and revert this when it loses focus.
    */
-  width: 115%;
+ 
```

Comment:
Unnecessary new-line, please revert.

---

Also, you've *still* included `package-lock.json` in this patch; in must be *completely* removed from the diff.

Reply:
I have removed the package-lock.json file. Do let me know if you want me to close this pull request and create a new one to clarify things. Alright, waiting for @timvandermeij 's review.

Reply:
Sorry, but you're *still* including the `package-lock.json` file and you've not [squashed the commit](https://github.com/mozilla/pdf.js/wiki/Squashing-Commits) either.
It really shouldn't be necessary to create a new PR, please just edit the existing commit series to address these issues.

Reply:
Hello, I have stashed the commits. And actually I have checked the "Load diff". It is showing me that package-lock.json was deleted.

Reply:
And, once more, please make sure that the *commit message* makes sense after you've squashed the commits (and edit it as required).

Reply:
Done :)